src/site/content/en/blog/why-coop-coep/index.md

```diff
@@ -86,7 +86,9 @@ successful.
 This all changed with
 [Spectre](https://en.wikipedia.org/wiki/Spectre_(security_vulnerability)), which
 makes any data that is loaded to the same browsing context group as your code
-potentially readable. If `evil.com` embeds a cross-origin image, they can use a
+potentially readable. By measuring the time certain operations take, attackers
+can guess the contents of the CPU caches, and through that, the contents of the
+process' memory. If `evil.com` embeds a cross-origin image, they can use a
```

Comment:
I'd add here: "Such timing attacks are possible with low-granularity timers that exist in the platform, but can be sped up with high-granularity timers, both explicit (like `performance.now()`) and implicit (like SharedArrayBuffers)"

Reply:
Thanks for the suggestion. Added.